packages/frontend/DEPOSIT_TEST_README.md

```
@@ -0,0 +1,107 @@
+
+3. **Verify contract addresses:**
+   - LendingPool: `0xFeE2CC90da44D83A4C1426d67EdD0F3b03d0204e`
+   - Gold Reserve Token: `0xd3871a7653073f2c8e4ed9d8d798303586a44f55`
+
+## Usage
+
+```bash
+node deposit-test.mjs
+```
+
+## What the script does:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)